javaparser-core/src/main/java/com/github/javaparser/Position.java

```diff
@@ -3,8 +3,8 @@
 import com.github.javaparser.ast.Node;
 
 public class Position {
-    private int line;
-    private int column;
+    public final int line;
```

Comment:
Well, I mean it is like a Point class. If we decide it is immutable then there is no real interest to have accessors.